### ISSUE TITLE
Support for JSON with comments

### ISSUE THREAD

**Issue by paddy74** [dangmai/vscode-workspace-default-settings#1]
`settings.json` supports JSON with C-style comments. Attempting to synchronize settings where either `settings.json` or `settings.default.json` contains a comment (`//` or `/* */`) results in the following error:

```
Command 'Workspace Default Settings: Synchronize Settings' resulted in an error (Unexpected token / in JSON at position 4)
```

This ticket requests that C-style comments according to JSON with comments, as supported by VSCode in `settings.json`, be added to this extension.

**Comment by dangmai** [comment]
Thanks for raising the issue, I am aware of this current limitation for the plugin - I'll see what I can do to implement it, but no guarantee that it will come out soon.

**Comment by dangmai** [comment]
I've published version `0.0.4` for this extension that supports JSON files with comments. Please note that the support only includes parsing those JSON files correctly and merging them, but does not include outputting the comments in the final settings file. Doing that is pretty complex, and I'd rather keep this extension as simple as possible. I'm happy to take PRs though if you're interested in implementing that.
